src/app/main.ts

```diff
@@ -67,18 +67,23 @@ if (!isDevEnv) {
 }
 
 /**
- * Main function that init the application
+ * Restarts the app in case of network issues
  */
-let oneStart = false;
-const startApplication = async () => {
-    await app.whenReady();
-    if (oneStart) {
-        return;
+const restartOnFirstInstall = async (): Promise<void> => {
+    const bootCount = config.getBootCount();
+    if (bootCount !== undefined && bootCount === 1) {
+        logger.warn(`Boot count fits the criteria of equal to 1, restarting the app`);
+        app.relaunch();
+        app.quit();
```

Comment:
Why do we restart the app first time? I completely understand the code, but we definetely need a comment (maybe with JIRA ticket nr) explaining why we do this.